planemo/options.py

```diff
@@ -134,7 +134,7 @@ def galaxy_python_version():
         '--galaxy_python_version',
         use_global_config=True,
         default=None,
-        type=click.Choice(['2', '2.7', '3', '3.5', '3.6', '3.7', '3.8']),
```

Comment:
To support older Galaxy releases, we may want to keep 2/2.7 here.

Reply:
True. Didn't think about this.